external/fv3fit/tests/emulation/thermobasis/test_emulator.py

```diff
@@ -0,0 +1,175 @@
+import argparse
+import datetime
+import numpy as np
+import tensorflow as tf
+from fv3fit.emulation.thermobasis.batch import to_dict
+from fv3fit.emulation.thermobasis.emulator import (
+    Config as Config,
+    Trainer,
+)
+from fv3fit.emulation.thermobasis.xarray import get_xarray_emulator
+from fv3fit.emulation.thermobasis.xarray import XarrayEmulator
+
+from fv3fit.emulation.thermobasis.loss import (
+    MultiVariableLoss,
+    RHLossSingleLevel,
+    QVLossSingleLevel,
+)
+from utils import _get_argsin
+import pytest
+from hypothesis import given
+from hypothesis.strategies import lists, integers
+
+
+def test_OnlineEmulator_partial_fit(state):
+    config = Config(batch_size=32, learning_rate=0.001, momentum=0.0, levels=63)
+
+    emulator = get_xarray_emulator(config)
+    emulator.partial_fit(state, state)
+
+
+def test_OnlineEmulator_partial_fit_logged(state):
+    config = Config(batch_size=8, learning_rate=0.0001, momentum=0.0, levels=63)
+    time = datetime.datetime.now().isoformat()
+
+    emulator = get_xarray_emulator(config)
+    writer = tf.summary.create_file_writer(f"logs_long/{time}")
+    with writer.as_default():
+        for i in range(10):
+            emulator.partial_fit(state, state)
```

Comment:
Do the logs need to be checked?  I'm guessing just having the writer available triggers the TBLogger

Reply:
It's a bit hard to check automatically, but the code is triggered.